oap-server/oal-rt/src/main/java/org/apache/skywalking/oal/rt/OALRuntime.java

```diff
@@ -150,6 +151,12 @@ public void start(ClassLoader currentClassLoader) throws ModuleStartException, O
             throw new ModuleStartException("load metrics functions error.", e);
         }
 
+        try {
+            FilterMatchers.INSTANCE.init();
```

Comment:
The `start` method is called more than once

Reply:
No side effect even initializing multiple times, fixed anyway